recipes/cgat-scripts/meta.yaml

```diff
@@ -1,179 +1,177 @@
     - bx-python # [py27]
-    - future ==0.16.0
-    - ggplot ==0.6.8
-    - jinja2 ==2.9.6
-    - matplotlib ==2.0.0
-    - networkx ==1.11
-    - nose ==1.3.7
-    - numpy ==1.12.1
-    - openpyxl ==2.4.0
-    - pandas ==0.19.2
-    - pep8 ==1.7.0
-    - psycopg2 ==2.7.1
-    - pybedtools ==0.7.9
-    - pysam ==0.10.0
-    - python-weblogo ==3.4
-    - pyyaml ==3.12
-    - rdflib ==4.2.2
-    - rpy2 ==2.7.8
-    - scipy ==0.19.0
-    - setuptools ==27.2.0
-    - six ==1.10.0
-    - scikit-learn ==0.18.1
+    - future 0.16.*
+    - ggplot 0.6.*
+    - jinja2 2.9.*
+    - matplotlib 2.0.*
+    - networkx 1.11.*
+    - nose 1.3.*
+    - numpy 1.12.*
+    - openpyxl 2.4.*
+    - pandas 0.20.*
+    - pep8 1.7.*
+    - psycopg2 2.7.*
+    - pybedtools 0.7.*
+    - pysam 0.11.1
+    - python-weblogo 3.4.*
+    - pyyaml 3.12.*
+    - rdflib 4.2.*
+    - rpy2 2.7.*
+    - scipy 0.19.*
+    - setuptools 27.2.*
+    - six 1.10.*
+    - scikit-learn 0.18.*
     # R deps
-    - r-base
+    - r-base 3.3.2
     - r-coloc
     - r-flashclust
-    - r-ggplot2 ==2.2.1
-    - r-gplots ==2.17.0
+    - r-ggplot2 2.2.*
+    - r-gplots 2.17.*
     - r-gridbase
-    - r-gridextra ==2.2.1
-    - r-gtools ==3.5.0
-    - r-maps ==3.1.0
-    - r-optparse ==1.3.2
-    - r-plyr ==1.8.4
-    - r-qqman ==0.1.2
+    - r-gridextra 2.2.*
+    - r-gtools 3.5.*
+    - r-maps
+    - r-optparse 1.3.*
+    - r-plyr 1.8.*
+    - r-qqman 0.1.*
     - r-rcolorbrewer
-    - r-reshape ==0.8.5
-    - r-reshape2 ==1.4.2
-    - r-samr ==2.0
-    - r-scales ==0.4.1
-    - r-sleuth ==0.28.0
+    - r-reshape 0.8.*
+    - r-reshape2 1.4.*
+    - r-samr 2.*
+    - r-scales 0.4.*
+    - r-sleuth 0.28.*
     - r-snow
-    - r-vegan ==2.3
-    - r-venndiagram ==1.6.16
-    - r-wgcna ==1.51
-    - bioconductor-biobase ==2.34.0
-    - bioconductor-biomart ==2.28.0
-    - bioconductor-deseq ==1.24.0
-    - bioconductor-deseq2 ==1.14.1
-    - bioconductor-edger ==3.16.5
-    - bioconductor-hilbertvis ==1.32.0
-    - bioconductor-ihw ==1.2.0
-    - bioconductor-limma ==3.30.13
-    - bioconductor-masigpro ==1.46.0
-    - bioconductor-medips ==1.24.0
-    - bioconductor-metagenomeseq ==1.16.0
-    - bioconductor-qvalue ==2.6.0
-    - bioconductor-rtracklayer ==1.32.2
-    - bioconductor-siggenes ==1.46.0
+    - r-vegan 2.3.*
+    - r-venndiagram 1.6.*
+    - r-wgcna 1.51.*
+    - bioconductor-biobase 2.34.*
+    - bioconductor-biomart 2.28.*
+    - bioconductor-deseq 1.24.*
+    - bioconductor-deseq2 1.14.*
+    - bioconductor-edger 3.16.*
+    - bioconductor-hilbertvis 1.32.*
+    - bioconductor-ihw 1.2.*
+    - bioconductor-limma 3.30.*
+    - bioconductor-masigpro 1.49.*
+    - bioconductor-medips 1.24.*
+    - bioconductor-metagenomeseq 1.16.*
+    - bioconductor-qvalue 2.6.*
+    - bioconductor-rtracklayer 1.34.*
+    - bioconductor-siggenes 1.46.*
 
   run:
     # system deps
-    - nomkl
-    - libgcc ==4.8.5
-    - zlib ==1.2.8
-    - libpng ==1.6.27
-    - mysqlclient ==1.3.9
+    - gcc 4.8.*
```

Comment:
@sebastian-luna-valero you have gcc as run time dependency this will not work I think. Please use libgcc. I would also not pin gcc to a specific version.